tests/test_multiprocessing.py

```diff
@@ -1,11 +1,15 @@
 from __future__ import print_function
 import os
+import threading
 import pytest
 import numpy as np
 from keras.models import Sequential
 from keras.layers.core import Dense
 from keras.utils.test_utils import keras_test
 
+gsteps_per_epoch = 100
```

Comment:
constants should be upper case

Reply:
Fixed in latest commit. Apologies for ignoring PEP 8 errs and warns in previous commit. This has been taken care of as well.